libs/barista-components/checkbox/BUILD.bazel

```
@@ -1,7 +1,32 @@
-load("//tools/bazel_rules:index.bzl", "stylelint")
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary", "sass_library")
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("//tools/bazel_rules:index.bzl", "jest", "ng_module_view_engine", "stylelint")
 
 package(default_visibility = ["//visibility:public"])
 
+ng_module_view_engine(
+    name = "compile",
+    srcs = glob(
+        include = ["**/*.ts"],
+        exclude = [
+            "**/*.spec.ts",
+            "src/test-setup.ts",
+        ],
+    ),
+    angular_assets = [
+        ":styles",
+        ":src/checkbox.html",
+    ],
+    module_name = "@dynatrace/barista-components/checkbox",
+    tsconfig = "tsconfig_lib",
+    deps = [
+        "//libs/barista-components/icon:compile",
+        "@npm//@angular/core",
+        "@npm//@angular/common",
+        "@npm//@angular/cdk/a11y"
```

Vendored dependencies under Bazel have no subpackes thats why the build is failing:

```
ERROR: /home/thebarista/barista/libs/barista-components/checkbox/BUILD.bazel:54:5: no such package '@npm//@angular/cdk/a11y': BUILD file not found in directory '@angular/cdk/a11y' of external repository @npm. Add a BUILD file to a directory to mark it as a package. and referenced by '//libs/barista-components/checkbox:test_compile'
```

Just specify the root package

```suggestion
        "@npm//@angular/cdk"
```